From 5ccb9eb82f66e3326e5493ba79ce1456a482165b Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Sat, 29 Dec 2012 23:35:58 +0100
Subject: [PATCH 03/20] ARM: mx5: Add CAN1 pinctrl data

This patch adds pinctrl data for different mux of CAN1 on MX53.

Signed-off-by: Marek Vasut <marex@denx.de>
Cc: Fabio Estevam <fabio.estevam@freescale.com>
Cc: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx53.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/imx53.dtsi b/arch/arm/boot/dts/imx53.dtsi
index 85973a1..8b78766 100644
--- a/arch/arm/boot/dts/imx53.dtsi
+++ b/arch/arm/boot/dts/imx53.dtsi
@@ -415,6 +415,13 @@
 							MX53_PAD_KEY_ROW2__CAN1_RXCAN 0x80000000
 						>;
 					};
+
+					pinctrl_can1_3: can1grp-3 {
+						fsl,pins = <
+							MX53_PAD_GPIO_7__CAN1_TXCAN 0x80000000
+							MX53_PAD_GPIO_8__CAN1_RXCAN 0x80000000
+						>;
+					};
 				};
 
 				can2 {
-- 
1.8.4.3

